reference/sql-query-tables/prices/README.md

```diff
@@ -1,9 +1,28 @@
 ---
-description: Tables containing historical prices of tokens
+description: Price data tables available in SQL query
 ---
 
 # Prices
 
+### Aggregate price base type tables available to query
+
+| View Name                                          | Description                                      |
+| -------------------------------------------------- | ------------------------------------------------ |
+| [`prices.trades`](trades.md)                       | Raw trade data from exchanges                    |
+| [`prices.trades_detailed`](trades\_detailed.md)    | Trade data with additional columns               |
+| [`prices.assets`](assets.md)                       | Tokens supported on Spice AI with metadata       |
+| [`prices.minute_resolution`](prices.md)                      | Per-minute price data for tokens pairs           |
```

Comment:
Need to confirm with team on naming `prices.minute_resolution`. I also thought `prices.ohlc_minute`